optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -457,6 +448,38 @@ def best_params(self) -> Dict[str, Any]:
             params.update(self.lgbm_params)
             return params
 
+    @property
+    def best_booster(self) -> lgb.Booster:
+        """Return the best booster."""
```

Comment:
```suggestion
        """Return the best booster.

        If the best booster cannot be found, :class:`ValueError` will be raised. To prevent the
        errors, please save boosters by specifying the `model_dir` arguments of
        :meth:`~optuna.integration.lightgbm.LightGBMTuner.__init__` when you resume tuning
        or you run tuning in parallel.
        """
```